content/docs/command-reference/config.md

```diff
@@ -179,11 +179,16 @@ This section contains the following options, which affect the project's
   > set.
 
 - `cache.shared` - permissions for newly created or downloaded cache files and
-  directories. The default is `0o664`(rw-r--r--) for files and `0o755`
-  (rwxr-xr-x) for directories. The only accepted value right now is `group`,
-  which makes DVC use `0o664` (rw-rw-r--) for files and `0o775` (rwxrwxr-x) for
-  directories, which is useful when you are using a a
-  [shared development server](/doc/use-cases/shared-development-server).
+  directories. The only accepted value right now is `group`, which makes DVC use
+  `664` (rw-rw-r--) for files and `775` (rwxrwxr-x) for directories. This is
+  useful when using a
+  [shared development server](/doc/use-cases/shared-development-server). The
+  default permissions for cache files is system dependent. In Linux and macOS
+  default permissions of cache is determined using
+  [`os.umask`](https://docs.python.org/3/library/os.html#os.umask). In Windows
+  due to the limitations of
+  [`os.chmod`](https://docs.python.org/3/library/os.html#os.chmod) only
+  read-only bit can be set.
```

Comment:
```suggestion
  default permissions for cache files is system dependent. In Linux and MacOS
  for example, they're determined using
  [`os.umask`](https://docs.python.org/3/library/os.html#os.umask).
```

Reply:
I removed the Windows part because I think that explanation wasn't accurate. It's OK, we can just skip it.